cpp/src/parquet/column_writer.cc

```diff
@@ -2048,8 +2058,7 @@ Status TypedColumnWriterImpl<Int32Type>::WriteArrowDense(
       WRITE_ZERO_COPY_CASE(DATE32, Date32Type, Int32Type)
       WRITE_SERIALIZE_CASE(DATE64, Date64Type, Int32Type)
       WRITE_SERIALIZE_CASE(TIME32, Time32Type, Int32Type)
-      WRITE_SERIALIZE_CASE(DECIMAL128, Decimal128Type, Int32Type)
-      WRITE_SERIALIZE_CASE(DECIMAL256, Decimal256Type, Int32Type)
```

Comment:
why not
```
      WRITE_SERIALIZE_CASE(DECIMAL32, Decimal32Type, Int32Type)
      WRITE_SERIALIZE_CASE(DECIMAL64, Decimal64Type, Int32Type)
      WRITE_SERIALIZE_CASE(DECIMAL128, Decimal128Type, Int32Type)
      WRITE_SERIALIZE_CASE(DECIMAL256, Decimal256Type, Int32Type)
```

Reply:
Perhaps we don't need `WRITE_SERIALIZE_CASE(DECIMAL64, Decimal64Type, Int32Type)`?

Reply:
> Perhaps we don't need 

I think we need, decimal64 is just decimal type but not limit the precision ?

Reply:
I'm open to this but suspicious of the value of it.